models/public/vitstr-small-patch16-224/README.md

```diff
@@ -0,0 +1,98 @@
+# vitstr-small-patch16-224
+
+## Use-case and high-level description
+
+The `vitstr-small-patch16-224` model is `small` version of the ViTSTR models.  ViTSTR is a simple single-stage model that uses a pre-trained Vision Transformer (ViT) to perform Scene Text Recognition (ViTSTR).  Small version of model requires an embedding size of 384 and number of heads of 6. Model is able to recognize alphanumeric text.
```

Comment:
I believe that model implementation requires this architecture, but we work with already designed model so
```suggestion
The `vitstr-small-patch16-224` model is `small` version of the ViTSTR models.  ViTSTR is a simple single-stage model that uses a pre-trained Vision Transformer (ViT) to perform Scene Text Recognition (ViTSTR).  Small version of model has an embedding size of 384 and number of heads of 6. Model is able to recognize alphanumeric text.
```

Reply:
and BTW, alphanumeric text means only alphabet and numbers, but as I can see from described symbols below, it also support additional symbols